### DISCUSSION TITLE
PayLoad website & Getting categories data

### DISCUSSION THREAD

**Discussion by Obadea** [payloadcms/payload#5564] [Q&A] [answered]
Is the payload site experiencing issues, or am I unable to access it? I've been attempting to access the documentation to learn how to retrieve category data. The issue is that when I try to obtain the categories data, it returns null. There don't appear to be any issues with the interface. If anyone knows a solution to this problem, please help. Thank you.

![code](https://github.com/payloadcms/payload/assets/124559546/c02bfefc-96bf-4784-80a3-066b933a7ac7)

![code2](https://github.com/payloadcms/payload/assets/124559546/3a0c782a-d97f-451d-9ec5-363b6c24d662)

**Comment by DanRibbens** [comment, marked as answer]
I'm not aware of any issues. You have categories saved that you can see in the admin UI?

It isn't clear to me from the code you shared how your frontend is fetching the categories. You can look at the network tab in the admin UI and how they are being retrieved from the list view for the collection.

**Reply by Obadea** [reply]
Thank you for your response. It turns out the issue was from my end — I mistakenly made changes to 'fetchDoc' instead of 'fetchDocs.' Now, I've encountered another problem. While I'm unsure if it's significant, I've attached an image of the issue below👇. Thanks again for your assistance.
![Screenshot (361)](https://github.com/payloadcms/payload/assets/124559546/034cbf92-8892-45fb-b937-04c8d3f67a3d)
